packages/core/src/Stores/client-store.js

```diff
@@ -451,9 +451,14 @@ export default class ClientStore extends BaseStore {
         );
 
         reaction(
-            () => [this.is_logged_in, this.is_authorize, this.is_passkey_supported],
+            () => [this.is_logged_in, this.is_authorize, this.is_passkey_supported, this.root_store.ui.is_mobile],
             () => {
-                if (this.is_logged_in && this.is_authorize && this.is_passkey_supported) {
+                if (
+                    this.is_logged_in &&
+                    this.is_authorize &&
+                    this.is_passkey_supported &&
+                    this.root_store.ui.is_mobile
```

Comment:
What is the plan when is_mobile gets deprecated? We are replacing these with hooks from @deriv-com/ui

Reply:
there is upcoming card to enable it for desktop, so we will remove this check soon